static/js/components/dashboard/CourseRow.js

```diff
@@ -14,6 +19,33 @@ export default class CourseRow extends React.Component {
     now: moment$Moment,
   };
 
+  courseMiddleSection: Function = (course: Course, now: moment$Moment): React$Element<*> => {
```

Comment:
Please move this logic into `CourseGrade`, and also rename `CourseGrade` and its file and the test file to `CoursePrice`